docs/Components/AttachedComponent.md

```diff
@@ -59,4 +59,10 @@ print(math.degrees(aileron.angle))
 plane.angle = math.radians(5)
 print(math.degrees(aileron.angle))
 >>> 14.99999999999
+```
+## ambient property
+This property returns the address of the environment in which the component is located
+```python
+print(wing.ambient)
+>>> adr.World.Ambient.Ambient object at 0x000001E4F70128E0
```

Comment:
Humn, in fact you found a bug Cassiano. It should return the ambient property, which is not defined in the FreeBody class.
I will open a PR working on that.
While we don't merge it we will put your PR on hold.